{"variants":[{"paths":["\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum\/rawrepresentable-implementations"],"traits":[{"interfaceLanguage":"swift"}]}],"sections":[],"schemaVersion":{"patch":0,"major":0,"minor":3},"identifier":{"url":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/RawRepresentable-Implementations","interfaceLanguage":"swift"},"metadata":{"role":"collectionGroup","title":"RawRepresentable Implementations","roleHeading":"API Collection","modules":[{"name":"FoundationX"}]},"kind":"article","hierarchy":{"paths":[["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError","doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum"]]},"topicSections":[{"identifiers":["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/hashValue"],"anchor":"Instance-Properties","generated":true,"title":"Instance Properties"},{"identifiers":["doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/hash(into:)"],"generated":true,"title":"Instance Methods","anchor":"Instance-Methods"}],"references":{"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/NetworkUnavailableReason-swift.enum/hash(into:)":{"url":"\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum\/hash(into:)","role":"symbol","type":"topic","kind":"symbol","title":"hash(into:)","abstract":[],"identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/hash(into:)","fragments":[{"text":"func","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"hash"},{"kind":"text","text":"("},{"kind":"externalParam","text":"into"},{"kind":"text","text":": "},{"kind":"keyword","text":"inout"},{"kind":"text","text":" "},{"kind":"typeIdentifier","text":"Hasher","preciseIdentifier":"s:s6HasherV"},{"text":")","kind":"text"}],"conformance":{"constraints":[{"code":"Self","type":"codeVoice"},{"type":"text","text":" conforms to "},{"type":"codeVoice","code":"Hashable"},{"text":" and ","type":"text"},{"type":"codeVoice","code":"RawValue"},{"text":" conforms to ","type":"text"},{"code":"Hashable","type":"codeVoice"},{"text":".","type":"text"}],"conformancePrefix":[{"type":"text","text":"Conforms when"}],"availabilityPrefix":[{"type":"text","text":"Available when"}]}},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError":{"title":"URLError","role":"symbol","url":"\/documentation\/foundationx\/urlerror","type":"topic","kind":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError","abstract":[{"text":"Describes errors in the URL error domain.","type":"text"}],"navigatorTitle":[{"text":"URLError","kind":"identifier"}],"fragments":[{"text":"struct","kind":"keyword"},{"kind":"text","text":" "},{"kind":"identifier","text":"URLError"}]},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/NetworkUnavailableReason-swift.enum/hashValue":{"type":"topic","fragments":[{"kind":"keyword","text":"var"},{"kind":"text","text":" "},{"kind":"identifier","text":"hashValue"},{"text":": ","kind":"text"},{"preciseIdentifier":"s:Si","text":"Int","kind":"typeIdentifier"}],"kind":"symbol","url":"\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum\/hashvalue","role":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum\/hashValue","title":"hashValue","abstract":[],"conformance":{"availabilityPrefix":[{"type":"text","text":"Available when"}],"conformancePrefix":[{"type":"text","text":"Conforms when"}],"constraints":[{"type":"codeVoice","code":"Self"},{"text":" conforms to ","type":"text"},{"code":"Hashable","type":"codeVoice"},{"type":"text","text":" and "},{"code":"RawValue","type":"codeVoice"},{"text":" conforms to ","type":"text"},{"code":"Hashable","type":"codeVoice"},{"text":".","type":"text"}]}},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX/URLError/NetworkUnavailableReason-swift.enum":{"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"NetworkUnavailableReason","kind":"identifier"}],"url":"\/documentation\/foundationx\/urlerror\/networkunavailablereason-swift.enum","kind":"symbol","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX\/URLError\/NetworkUnavailableReason-swift.enum","role":"symbol","title":"URLError.NetworkUnavailableReason","abstract":[{"text":"Reasons used by URLError to indicate that a URLSessionTask failed because of unsatisfiable network constraints.","type":"text"}],"navigatorTitle":[{"kind":"identifier","text":"NetworkUnavailableReason"}]},"doc://com.swallow.documentation.FoundationX/documentation/FoundationX":{"abstract":[],"kind":"symbol","title":"FoundationX","identifier":"doc:\/\/com.swallow.documentation.FoundationX\/documentation\/FoundationX","url":"\/documentation\/foundationx","role":"collection","type":"topic"}}}